gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -364,6 +364,24 @@ message TripUpdate {
     extensions 9000 to 9999;
   }
   optional TripProperties trip_properties = 6;
+  
+  // Realtime occupancy immediately after departure for a given stop on a
```

Comment:
Is there a reason to allow data for past events?

I'd be inclined to have stronger language that requires this to be a predicted future value immediately after departure for the given stop. One sticking point for predictions currently, especially surrounding TripUpdate.delay, is that it's not clear if the value is observed fact or a prediction about the future. I'd prefer to have stricter guidance here to know exactly what we're seeing in feeds. It seems to me if you just want real-time or past info, you should be looking at VehiclePosition.occupancy_status. This new field should include some predictive intelligence that presumably adds value over the data in VehiclePosition.

Reply:
The reason was only consistency with other existing fields of TripUpdate. But it would be fine to change to "future only", as you mention.
That would imply consistency with VehiclePosition.current_stop_sequence, or allow current_stop_sequence to be inferred if it isn't set.